test/sys/test_wait.rs

```diff
@@ -37,6 +37,19 @@ fn test_wait_exit() {
     }
 }
 
+#[test]
+fn test_waitstatus_pid() {
+    let _m = ::FORK_MTX.lock().expect("Mutex got poisoned by another test");
+
+    match fork().unwrap() {
+        Child => {},
```

Comment:
You mean that the test harness executes the function as a normal function, so the forked process will duplicate what test harness does?

Why `_exit` and not `std::process::exit`? The former is `unsafe`.